### PR TITLE
add fibonacci.py and reverse.py in fibonacci and reverse_a_number res…

### PR DESCRIPTION
**Pull request by DamiyanVC**
…pectively

### CODE REVIEW

File: fibonacci/fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+num=int(input("Enter the number of elements of the series:"))
```
Comment (t2013anurag) [inline fibonacci/fibonacci.py]: use proper spacing b/w operands and operators.

File: reverse_a_number/reverse.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#python program to reverse a number
+
```
Comment (t2013anurag) [inline reverse_a_number/reverse.py]: remove extra line

File: fibonacci/fibonacci.py (hunk 1 above)
Comment (DamiyanVC) [reply fibonacci/fibonacci.py]: ok..
modified it

File: reverse_a_number/reverse.py (hunk 2 above)
Comment (DamiyanVC) [reply reverse_a_number/reverse.py]: done

File: fibonacci/fibonacci.py (hunk 1 above)
Comment (DamiyanVC) [reply fibonacci/fibonacci.py]: ok

File: fibonacci/reverse.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Number = int(input("Please Enter any number:"))
```
Comment (t2013anurag) [inline fibonacci/reverse.py]: remove this file.

**Pull request merged**